aksel.nav.no/README.md

```diff
@@ -41,7 +39,7 @@ yarn serve:next
 
 **Rettigheter**
 
-Sanity-datasettet er privat, noe som betyr du må ha tilgang til sanity applikasjonen + følgende token for å få data lokalt:
+Sanity-datasettet er privat, noe som betyr du må ha tilgang til Sanity-applikasjonen + følgende tokens for å få data lokalt:
```

Comment:
This contradicts the sentence bellow where it says you only need SANITY_READ_NO_DRAFTS. Please help me rewrite this section.